vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -586,32 +595,26 @@ If you use the image URL hash `icon-top`, the image will be styled according to
           title: Filters and parameters
         ```
     === "Result"
-           [![NavCardIcon]][NavCardIcon]
-
-       [NavCardIcon]: ../../assets/user_guides/components/nav_card_icon.png
-
-Note that in the above example the first [`Card`][vizro.models.Card] navigates to an existing [`Page`][vizro.models.Page]
-in the app with `path = filters-and-parameters` and the second one to an external link.
+           [![NavCard]][NavCard]
 
-#### Add an icon responsive to theme switch
+       [NavCard]: ../../assets/user_guides/components/nav_card.png
 
-To add an icon that is responsive to the theme switch, you will need to override the value of the [`filter` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/filter).
+If you now click on the card area, you should automatically be redirected to the relevant `href`.
 
-The `filter` CSS property lets you add visual effects to elements using different functions. In our example, we're using the `--inverse-color` CSS variable from the Vizro theme.
+???+ note
 
-It uses the  CSS `invert()` function to flip the color of the icon when you switch themes. Note that this only works if your initial icon has a white fill color. If your icon is not white, you can change its color by adding `fill="white"` to the SVG code.
+    When using the [`Card`][vizro.models.Card], keep the following considerations in mind:
 
-Assign the predefined CSS variable `--inverse-color` to the `filter` property of your selected icon.
+    - If the href given is a relative URL, it should match the `path` of the [`Page`][vizro.models.Page] that the [`Card`][vizro.models.Card] should navigate to.
+    - If the href given is an absolute link, it should start with `https://` or an equivalent protocol.
 
-```css
-img[src*="#my-image"] {
-  filter: var(--inverse-color);
-}
-```
 
-??? example "Responsive icon"
-    ![responsive icon](../../assets/user_guides/components/responsive_icon.gif)
+### Create a KPI card
+To create a KPI card, you can use the existing KPI card functions from [`vizro.figures`](../API-reference/figure-callables.md).
+Unlike the static text card `vm.Card`, a KPI card must be created using a figure function.
+This allows the text content of the KPI card to dynamically update based on any changes in controls or actions.
 
+For detailed examples on how to create a KPI card, please refer to the [figure user guide on KPI cards](figure.md#create-kpi-cards).
```

Comment:
Does this split-up make sense to you? Or would you rather like to see the code for the kpi cards in the card-button.md instead of the figure.md?

Reply:
This is probably a query for @antonymilne rather than me, and I'm happy to see this in situ here for now.

Reply:
I think this is perfect how you have it.